src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/dist/index.html

```diff
@@ -4,28 +4,24 @@
   <head>
     <meta charset="UTF-8">
     <title>[[SWAGGER_UI_TITLE]]</title>
-    <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
-    <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
+    <link rel="icon" type="image/png" href="[[SWAGGER_UI_FAVICON_URL]]" sizes="32x32" />
```

Comment:
Good approach! But we need to consider those things:

* We don't know whether the favicon's MIME type is `image/png` or `image/ico` or `image/jpg` or something else.
* We also don't know the favicon's size either.
* We also don't know how many favicons will be defined, with different sizes.

Therefore, I would suggest the entire HTML tag, `<link .../>` needs to be replaced with a placeholder `[[SWAGGER_UI_FAVICON_LINKS]]`, and change other logics in accordance with this.

Reply:
Thank you for the fast code review! Based on the comments, I'll PR it again.😀